spring-cloud-function-context/pom.xml

```diff
@@ -25,6 +25,10 @@
 			<version>${spring-cloud-function.version}</version>
 		</dependency>
 		<dependency>
+			<groupId>org.springframework</groupId>
```

Comment:
ok, fine for now... at some point down the road, we should explore options for reducing dependencies in general